src/App/App.js

```diff
@@ -459,7 +459,7 @@ function FullApp() {
               <Route exact path="/buy">
                 <Buy savedSlippageAmount={settings.savedAllowedSlippage} setPendingTxns={setPendingTxns} />
               </Route>
-              <Route exact path="/pools">
+              <Route exact path="/pools/:market?/:operation?/:mode?">
```

Comment:
we don't change page content based on the params righs? it just select some things in TradeBox. For that case usually query params is used instead of path params because path params is structured and if we will add more params then old links will be invalid

Reply:
also with query params is more flexible, imagine that we want to post some link in twitter post to suggest users open Shorts and don't want to specify a market